slash/utils/traceback_proxy.py

```diff
@@ -176,3 +176,8 @@ def create_traceback_proxy(tb=None, frame_correction=0):
                 current = current.tb_next
 
         return (first, current)
+
+    def istraceback(object):
+        return isinstance(object, (types.TracebackType, TracebackProxy))
+
+    inspect.istraceback = istraceback  # so that it cooperates with 'inspect.getframeinfo'
```

Comment:
avoid patching existing modules. You can achieve the same goal with a `__class__` property on TracebackProxy

Reply:
you learn something new every day...:)